Sources/X509/Verifier/Verifier.swift

```diff
@@ -120,7 +120,7 @@ public struct Verifier<Policy: VerifierPolicy> {
                     )
                 )
 
-                for parent in intermediateParents {
+                for parent in intermediateParents.reversed() {
```

Comment:
Can we document why we reverse this? (Answer: we push these onto a stack so the order in which we iterate them here is the _reverse_ order in which we explore them).